packages/light-client/README.md

```diff
@@ -57,16 +57,19 @@ npm i -S @chainsafe/blst
 ```
 
 ```ts
-import {getClient} from "@lodestar/api";
-import {createChainForkConfig} from "@lodestar/config";
-import {networksChainConfig} from "@lodestar/config/networks";
 import {Lightclient, LightclientEvent} from "@lodestar/light-client";
 import {LightClientRestTransport} from "@lodestar/light-client/transport";
-import {getFinalizedSyncCheckpoint, getGenesisData, getLcLoggerConsole} from "@lodestar/light-client/utils";
-
-const config = createChainForkConfig(networksChainConfig.mainnet);
-const logger = getLcLoggerConsole({logDebug: Boolean(process.env.DEBUG)});
-const api = getClient({urls: ["https://beacon-node.your-domain.com"]}, {config});
+import {
+  getFinalizedSyncCheckpoint,
+  getGenesisData,
+  getConsoleLogger,
+  getApiFromUrl,
+  getChainForkConfigFromNetwork,
+} from "@lodestar/light-client/utils";
+
+const config = getChainForkConfigFromNetwork("mainnet");
+const logger = getConsoleLogger({logDebug: Boolean(process.env.DEBUG)});
+const api = getApiFromUrl({urls: ["https://lodestar-mainnet.chainsafe.io"]}, {config});
```

Comment:
Same question about using our real URL

Reply:
I wanted to have an example which people can run by just copying the code. We have similar pattern in the prover already but pointing to sepolia url.

Reply:
I think that is a fine idea.... just not our mainnet nodes